Doc/whatsnew/3.11.rst

```diff
@@ -176,14 +176,33 @@ The :option:`-X` ``no_debug_ranges`` option and the environment variable
 See :pep:`657` for more details. (Contributed by Pablo Galindo, Batuhan Taskaya
 and Ammar Askar in :issue:`43950`.)
 
+Exception Groups and ``except*`` (PEP 654)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:pep:`654` introduces language features that enable a program
+to raise and handle multiple unrelated exceptions simultaneously.
+The builtin types :exc:`ExceptionGroup` and :exc:`BaseExceptionGroup`
+make it possible to group exceptions and raise them together.
+The new ``except*`` syntax generalizes ``except`` to match subgroups
```

Comment:
```suggestion
make it possible to group exceptions and raise them together,
and the new :keyword:`except*` syntax
generalizes :keyword:`except` to match subgroups
```

Use the appropriate semantic roles here, and link them.

Also, it currently reads rather choppy; a comma instead of a period (full stop) improves flow somewhat